src/data-sources/components/AddDataSourceDropdown.tsx

```diff
@@ -50,6 +50,11 @@ export const AddDataSourceDropdown = () => {
 							label: SUPPORTED_SERVICES_LABELS.shopify,
 							value: 'shopify',
 						},
+						{
+							icon: HttpIcon,
```

Comment:
Note we're still using the `HttpIcon` here instead of something more specific.

Reply:
I pushed [a commit](https://github.com/Automattic/remote-data-blocks/pull/228/commits/aa9a98d9e7824c4e986cbcfee8eb8078e111a12f) that includes an icon, however the icon is licensed under the MIT license so we should get it replaced with something custom probably.

Reply:
The icon came from [here](https://www.svgrepo.com/svg/314577/salesforce).

Reply:
I switched to the icon provided by Salesforce in their [brand guide](https://brand.salesforce.com), though it isn't sizing well for our use case. You might be able to try again. I had to remove a bunch of stuff from the SVG contents to get it to work at all.